### PR TITLE
Updated the dataless information to work off of webservices.

### PR DESCRIPTION
**Pull request by aringler-usgs**
Switched to webservices.  I also removed a bunch of old dataless parsing routines.

### CODE REVIEW

File: accelcomp.py [hunk 1]
```diff
@@ -540,15 +296,34 @@ def writestats(statfile,streamin,comp):
...
 
 
-#Read in the CMT solution 
-cmtfile = parserval.cmt
+# Read in the CMT solution from the synthetic directory
 if debug:
-    print("We are using local synthetics")
-if not os.path.isfile(cmtfile):
-    print("No CMT found")
+    print "We are using local synthetics"
```
Comment (jholland-usgs) [inline accelcomp.py]: Preference for python3 compatible print("statements")

File: accelcomp.py [hunk 2]
```diff
@@ -612,86 +384,44 @@ def writestats(statfile,streamin,comp):
...
+            tr.taper(max_percentage=0.05, type='cosine')
+            #If we have an accelerometer we want an extra zero to go to displacement
+            tr.remove_response(inventory = inv, output='DISP')
+            #Here we filter, integrate, taper, trim, detrend, and filter
+            tr.filter("bandpass",freqmin = userminfre,freqmax= usermaxfre, corners=filtercornerpoles)
+            tr.taper(max_percentage=0.05, type='cosine')
+            tr.trim(starttime=eventtime + tshift/2,endtime=(eventtime+lents + tshift/2))
+            tr.detrend()
+            tr.filter("bandpass",freqmin = userminfre,freqmax= usermaxfre, corners=filtercornerpoles)
         #except:
```
Comment (jholland-usgs) [inline accelcomp.py]: Please remove commented code when possible. Thanks

Comment (aringler-usgs) [reply accelcomp.py]: Fixed this (I removed it for testing).

File: accelcomp.py (hunk 1 above)
Comment (aringler-usgs) [reply accelcomp.py]: Fixed.

**Pull request merged**